### PR TITLE
Fix environment bug,version of golang should be 1.4+, add reminding i…

### PR DESCRIPTION
**Pull request by zenlinTechnofreak**
Fix environment bug,version of golang should be 1.4+, add reminding in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ Hence we are proposing this specification, along with an initial set of plugins
 
 ## How do I use CNI?
 
+## Prepare Job
```
Comment (jonboulle) [inline README.md]: Would you mind changing this to the following?
```
## Requirements

CNI requires Go 1.4+ to build.

Comment (zenlinTechnofreak) [inline README.md]: yes,  am away of my computer now, I am going to commit it one hour later.THX

**Comment by zenlinTechnofreak** [comment]
@jonboulle I have modified the commit in this pull request, plz deal with it. Thx.

**Comment by jonboulle** [comment]
thanks!

**Pull request merged**